Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -343,6 +343,19 @@ private func rewardsCollectionViewController(
   let rewardsCollectionViewController = RewardsCollectionViewController
     .instantiate(with: project, refTag: refTag, context: .createPledge)
 
+  let closeButton = UIBarButtonItem(
```

Comment:
The closeButton is set at the point of presentation now so that we don't see it when pushing onto the navigation controller stack.